src/components/LayoutHeader/Nav/styles.module.css

```diff
@@ -15,7 +15,7 @@
   z-index: 1;
   padding: 16px 24px;
   font-weight: 500;
-  font-size: 18px;
+  font-size: 16px;
```

Comment:
I went ahead and lowered the font for the "Get Started" button. If this needs to be decreased again, let me know!